generic-database-plugin/widgets/Database-action.json

```diff
@@ -84,4 +76,4 @@
       ]
     }
   ]
-}
+}
```

Comment:
files should end with a newline (same for other widgets)

Reply:
fixed